### PR TITLE
VACMS-7321: Add "Enforce unique combo office" field to fix duplicated Non-clinical Service

### LINKED ISSUE #7321: Duplicate Non-clinical Service should not be allowed at the same location
## User Story or Problem Statement

As an editor I should not be allowed to create two of the same kind of Non-clinical Service
for the same location so that I do not accidentally create duplicates, which are confusing and would result in needing to delete one.

image

## Acceptance Criteria
- [x] vha_facility_nonclinical_service contains a field of type `allow only one`
   - [x] use existing: field_enforce_unique_combo_offic
   - [x]  Label "Enforce unique combo office"
   - [x] Unique field combination field_facility_location and field_service_name_and_descripti
   - [x] Should manage form display right above Service
   
image

- [x] I am prevented from creating two Non-clinical services of the same type at the same facility.
- [x] Spec tool tests are updated to account for the new field.

## Design principles

Veteran-centered
- [ ] `Single source of truth`: Increase reliability and consistency of content on VA.gov by providing a single source of truth.
- [ ] `Accessible, plain language`: Provide guardrails and guidelines to ensure content quality.
- [ ] `Purposely structured content`: Ensure Content API can deliver content whose meaning matches its structure.
- [ ] `Content lifecycle governance`: Produce tools, processes and policies to maintain content quality throughout its lifecycle.

Editor-centered
- [ ] `Purpose-driven`: Create an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [x] `Efficient`: Remove distractions and create clear, straightforward paths to get the job done.
- [ ] `Approachable`: Offer friendly guidance over authoritative instruction.
- [ ] `Consistent`: Reduce user’s mental load by allowing them to fall back on pattern recognition to complete tasks.
- [ ] `Empowering`: Provide clear information to help editors make decisions about their work.

### CMS Team

Please leave only the team that will do this work selected. If you're not sure, it's fine to leave both selected.

- [ ] `Platform CMS Team`
- [x] `Sitewide CMS Team`

### PR DESCRIPTION
**Pull request by netboss**
- VACMS-7321:
    - Added new "Enforce unique combo office" (field_enforce_unique_combo_offic) field to the "VAMC Facility Non-clinical Service" content type and configured it.
    - Adjusted the "VAMC Facility Non-clinical Service" content type form display and rearranged the new field.
    - Updated "content_model_bundles.feature" and "content_model_vamc_content_type_fields.feature" Behat features.

## Description

closes #7321 

## Testing done

Passes all tests

## Screenshots

1. Existing **VAMC Facility Non-clinical Service** node: **Billing and insurance - Sheridan VA Medical Center**
existing-VAMC-facility-non-clinical-node

1. Duplicated **VAMC Facility Non-clinical Service** node blocked on save: 
new-VAMC-facility-non-clinical-node-save-notice

## QA steps

As user administrator
1. Go to a **VAMC Facility Non-clinical Service** node e.g.: `/node/42145/edit` and use this node's data 
2. Create another node **VAMC Facility Non-clinical Service** `/node/add/vha_facility_nonclinical_service` with the same data and click save
3. After trying to save the "duplicated" node you should see the **_There is existing content: Billing and insurance - Sheridan VA Medical Center, with identical values for the following field(s): Facility, Service._** message (see screen shot # 2 above)

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS team`
- [ ] `Sitewide CMS Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: config/sync/core.entity_form_display.node.vha_facility_nonclinical_service.default.yml [hunk 1]
```diff
@@ -36,9 +38,8 @@ third_party_settings:
     group_editorial_workflow:
       children:
         - moderation_state
-        - revision_log
```
Comment (swirtSJW) [inline config/sync/core.entity_form_display.node.vha_facility_nonclinical_service.default.yml:39]: You need to add this one back by hand.   It is the cause of the failing test.

**Review by swirtSJW — CHANGES_REQUESTED** [review]

**Review by swirtSJW — APPROVED** [review]
Nice work @netboss

**Comment by netboss** [comment]
@swirtSJW 
What else is needed here? Or if this already approved and ready to be merged? Please let me know.

**Comment by swirtSJW** [comment]
image

**Pull request merged**